static/src/server.ts

```diff
@@ -124,20 +74,6 @@ const CHAR_WIDTHS = [
 const CHAR_AVG_WIDTH = 5.0341796875;
 // Height of a 10px Roboto character.
 const CHAR_HEIGHT = 13;
-// Height of the svg to render.
```

Comment:
move line 57-76 to constants.ts as well?

Reply:
I kept those constants in server because they're only used in server.ts for mocking the getBBox functions ..